apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -82,19 +82,24 @@ export class SettingsCog extends Component {
   closeModelManager = () => this.setState({managingModels: false});
   handleClickOutside = () => this.close();
 
+  getPageConstants() {
+    return getStore().getState().pageConstants;
+  }
   areLibrariesEnabled() {
-    let pageConstants = getStore().getState().pageConstants;
-    return pageConstants && pageConstants.librariesEnabled;
+    return this.getPageConstants()?.librariesEnabled;
   }
 
   areAIToolsEnabled() {
-    let pageConstants = getStore().getState().pageConstants;
-    return pageConstants && pageConstants.aiEnabled;
+    return this.getPageConstants()?.aiEnabled;
+  }
+
+  isCurriculumLevel() {
+    return this.getPageConstants()?.isCurriculumLevel;
```

Comment:
what happens in labs where `isCurriculumLevel` isn't defined?

Reply:
In labs with a `SettingsCog` for which `isCurriculumLevel` isn't defined such as Game Lab, the `showMakerToggle` will be assigned `false` so that this function is not called [within the `render` function](https://github.com/code-dot-org/code-dot-org/blob/19cf8adf124303c70de78f5a18be32f583704444/apps/src/lib/ui/SettingsCog.jsx#L136). 

Out of curiosity, I swapped the values of `this.props.showMakerToggle` and `!this.isCurriculumLevel()` just to see what would be returned, and `undefined` is returned. Checked locally on Game Lab, and only two options are displayed when the settings cog is clicked: 'Manage Assets' and 'Manage Libraries'.